mysql-test/t/acl_mdl.test

```diff
@@ -0,0 +1,41 @@
+--source include/have_debug_sync.inc
+--source include/have_debug.inc
+--source include/count_sessions.inc
+
+--echo #
+--echo # PS-7143 : Server freezes and reaches max connections due to ACL Cache Lock waits
+--echo #
+
+--echo # This test will ensure that when deadlock checks are done for waiting lock, it
+--echo # cannot be ACL Lock. A debug assertion will fail
```

Comment:
This makes the reader to think that the following test will cause an assertion failure, which is wrong. IMO, it is better to remove the last sentence from the line and just keep the rest as is.